autoblocks/_impl/util.py

```diff
@@ -51,4 +57,4 @@ async def sem_coro(coro: Coroutine[Any, Any, Any]) -> Any:
 
     # return_exceptions=True causes exceptions to be returned as values instead
     # of propagating them to the caller. this is similar in behavior to Promise.allSettled
-    await asyncio.gather(*(sem_coro(c) for c in coroutines), return_exceptions=True)
+    return await asyncio.gather(*(sem_coro(c) for c in coroutines), return_exceptions=True)
```

Comment:
i thought this `return await` didn't work, did something change?

Reply:
We were mislead - this wasn't working because we were mocking time in the unit test that this was getting hung up on. 


Issue as related to this: https://github.com/spulec/freezegun/pull/470

Reply:
ooh okay